www/source/blog/2018-02-07-openshift-support.html.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Habitat Can Now Run in OpenShift
+date: 2018-02-07
+author: Christopher Maier
+tags: blog, supervisor, packaging, openshift
+category: supervisor
+classes: body-article
+---
+
+With the 0.53.0 release of Habitat, we're happy to announce that the users can now run their Habitat-exported Docker containers on the [OpenShift](https://openshift.com) platform! In this post we'll show you what you need to do to run your own apps in OpenShift, as well as what prevented Habitat from running on OpenShift until now, and how we solved this particular issue.
```

Comment:
`that the users` -> `that users`

Reply:
I think you can lose the last comma `until now, and` since the comma precedes an incomplete clause